src/routes/v2/authenticated/review.ts

```diff
@@ -634,6 +635,61 @@ export class ReviewsRouter {
     return res.status(200).send()
   }
 
+  getComments: RequestHandler<
+    { siteName: string; requestId: number },
+    CommentItem[],
```

Comment:
type needs to be `CommentItem[] | ResponseErrorBody` so that `{message: "<some_message>"}` passes type check

Reply:
whoops, fixed in https://github.com/isomerpages/isomercms-backend/pull/534/commits/d60c399848f15ddc5465da95e048103f03902053